lib/puppet/provider/postgresql_conf/ruby.rb

```diff
@@ -74,7 +74,7 @@ def write_config(file, lines)
   # check, if resource exists in postgresql.conf file
   def exists?
     select = parse_config.select { |hash| hash[:key] == resource[:key] }
-    raise ParserError, "found multiple config items of #{resource[:key]} found, please fix this" if select.length > 1
+    raise Puppet::Error, "found multiple config items of #{resource[:key]}, please fix this" if select.length > 1
```

Comment:
FWIW, providers can call the `fail` method which will raise `Puppet::Error`, for example see the base exec provider: https://github.com/puppetlabs/puppet/blob/fce49baa3767fffb000f1b26d3a7b4edbec58999/lib/puppet/provider/exec.rb#L105

Reply:
I just wanted to fix a simple typo. I don't know how to properly write tests for the `fail()` method. If someone wants to do this: feel free.